programs/compute-budget/src/lib.rs

```diff
@@ -1,6 +1,7 @@
 use solana_program_runtime::declare_process_instruction;
 
-pub const DEFAULT_COMPUTE_UNITS: u64 = 150;
+// compute budget instructions are processed before VM, therefore do not have execution cost
+pub const DEFAULT_COMPUTE_UNITS: u64 = 0;
```

Comment:
Can't do this yet -- the idea of not consume CU works, but it'd trigger `BuiltinProgramsMustConsumeComputeUnits` error